File: backend/src/main/java/io/kongkham/kcurr/ApiClients/CurrencyBeacon/CurrencyBeaconCurrCountriesApiResponseResponse.java
package io.kongkham.kcurr.ApiClients.CurrencyBeacon;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;

@Data
public class CurrencyBeaconCurrCountriesApiResponseResponse {
    @JsonProperty("name")
    private String currNameWithCountryName;
    @JsonProperty("short_code")
    private String currCode;
    @JsonProperty("symbol")
    private String currSymbol;
}

File: backend/backend/ApiClients/CurrencyBeacon/CurrencyBeaconCurrCountriesApiResponseResponse.cs
using System.Text.Json.Serialization;

namespace backend.ApiClients.CurrencyBeacon;

public class CurrencyBeaconCurrCountriesApiResponseResponse
{
    [JsonPropertyName("name")]
    public string CurrNameWithCountryName { get; set; }

    [JsonPropertyName("short_code")]
    public string CurrCode { get; set; }

    [JsonPropertyName("symbol")]
    public string CurrSymbol { get; set; }
}